src/frontegg-auth/Cargo.toml

```diff
@@ -14,6 +14,8 @@ jsonwebtoken = "8.2.0"
 mz-ore = { path = "../ore", features = ["network"] }
 mz-sql = { path = "../sql" }
 reqwest = { version = "0.11.13", features = ["json"] }
+reqwest-middleware = { version = "0.2.3", git = "https://github.com/ParkMyCar/reqwest-middleware.git", branch = "disable-wasm-deps" }
```

Comment:
I have no idea if we have a policy or not around using non-mz orgs/users when forking like this. Should this instead be under the mz org? Can you investigate?

Reply:
I think the general strategy is to make MZ owned forks, I started with a personal fork just to validate it would work. I asked in Slack if someone could make one!

Reply:
Made the fork and update this dep